packages/protocol/deploy-configs/ethereum.json

```diff
@@ -194,7 +194,25 @@
       "liqRatio": 820000000000000000,
       "maxLtv": 780000000000000000,
       "name": "BorrowingVault-WETHUSDC-3",
-      "providers": ["Morpho_Compound", "Morpho_Aave_V2"],
+      "providers": ["Morpho_Aave_V2","Morpho_Compound","Compound_V3"],
```

Comment:
formatting: space after the comma

Reply:
I missed that, though I just ran the standard json formatting and it breaks the providers like this:
```json
{
      "collateral": "WETH",
      "debt": "USDC",
      "liqRatio": 820000000000000000,
      "maxLtv": 780000000000000000,
      "name": "BorrowingVault-WETHUSDC-3",
      "providers": [
        "Morpho_Aave_V2",
        "Morpho_Compound",
        "Compound_V3"
      ],
      "rating": 85
    },
```
Would this be ok?